code/game/objects/items/weapons/dice.dm

```diff
@@ -74,20 +127,20 @@
 	update_icon()
 	if(initial(icon_state) == "d00")
 		result = (result - 1)*10
+	if(special_faces.len == sides)
+		result = special_faces[result]
 	if(user != null) //Dice was rolled in someone's hand
-		user.visible_message("<span class='notice'>[user] has thrown [src]. It lands on [result]. [comment]</span>", \
+		user.visible_message("[user] has thrown [src]. It lands on [result]. [comment]", \
 							 "<span class='notice'>You throw [src]. It lands on [result]. [comment]</span>", \
-							 "<span class='notice'>You hear [src] landing on [result]. [comment]</span>")
+							 "<span class='italics'>You hear [src] rolling, it sounds like a [fake_result].</span>")
 	else if(src.throwing == 0) //Dice was thrown and is coming to rest
 		visible_message("<span class='notice'>[src] rolls to a stop, landing on [result]. [comment]</span>")
-/*
+
 /obj/item/weapon/dice/d4/Crossed(var/mob/living/carbon/human/H)
 	if(istype(H) && !H.shoes)

```

Comment:
> all of these dice changes are insanely non-atomic

I'm pretty sure the dice changes are because survival shelters can contain dice, but our dice aren't literally identical to TG's, so, uh, they clearly need to be updated? ¯\\\_(ツ)_/¯